pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/documentation/CommentSizeRule.java

```diff
@@ -67,12 +67,11 @@ public Object visit(ASTCompilationUnit cUnit, Object data) {
             int offset = comment.getBeginLine();
             for (int lineNum : lineNumbers) {
                 int lineNumWithOff = lineNum + offset;
-                addViolationWithMessage(
-                    data,
+                asCtx(data).addViolationWithPosition(
                     cUnit,
-                    this.getMessage() + ": Line too long",
                     lineNumWithOff,
-                    lineNum
+                    lineNumWithOff,
```

Comment:
This is a bug fix btw. This used to fail because the lines are not ordered. It is missing a test